webapp/src/components/intersections/DashboardPage.tsx

```diff
@@ -66,32 +66,120 @@ const Page = () => {
+      lg: computeWidth(minWidth, muiBreakpoints.lg, gridWidth),
+      xl: computeWidth(minWidth, muiBreakpoints.xl, gridWidth),
+    }
+  }
+
+  const widthPerBar = 22
+  const legendPadding = 100
+  const minChartWidth = 200
+
+  const getChartWidth = (numBars: number) => Math.max((numBars ?? 0) * widthPerBar + legendPadding, minChartWidth)
+  const connectionOfTravelMinWidth = useMemo(
+    () => getChartWidth(connectionOfTravelAssessment?.connectionOfTravelAssessmentGroups?.length),
+    [connectionOfTravelAssessment]
+  )
+  const stopLineStopMinWidth = useMemo(
+    () => getChartWidth(stopLineStopAssessment?.stopLineStopAssessmentGroup?.length),
+    [stopLineStopAssessment]
+  )
+  const signalStateEventMinWidth = useMemo(
+    () => getChartWidth(signalStateEventAssessment?.signalStateEventAssessmentGroup?.length),
+    [signalStateEventAssessment]
+  )
+
+  // Lane direction of travel shows 2 bars per unique lane ID
+  const laneDirectionOfTravelMinWidth = useMemo(() => {
+    const laneDirectionOfTravelNumUniqueLanes =
+      new Set(laneDirectionOfTravelAssessment?.laneDirectionOfTravelAssessmentGroup?.map((group) => group.laneID) ?? [])
+        .size * 2
+    return getChartWidth(laneDirectionOfTravelNumUniqueLanes)
+  }, [laneDirectionOfTravelAssessment])
+
+  const connectionOfTravelBreakpoints = useMemo(
+    () => generateBreakpoints(connectionOfTravelMinWidth + chartCardPadding),
+    [connectionOfTravelMinWidth]
+  )
+  const stopLineStopBreakpoints = useMemo(
+    () => generateBreakpoints(stopLineStopMinWidth + chartCardPadding),
+    [stopLineStopMinWidth]
+  )
+  const signalStateEventBreakpoints = useMemo(
+    () => generateBreakpoints(signalStateEventMinWidth + chartCardPadding),
+    [signalStateEventMinWidth]
+  )
+  const laneDirectionOfTravelBreakpoints = useMemo(
+    () => generateBreakpoints(laneDirectionOfTravelMinWidth + chartCardPadding),
+    [laneDirectionOfTravelMinWidth]
+  )
+
   return (
     <>
       <Box
         component="main"
         sx={{
           flexGrow: 1,
           py: 8,
+          width: '100%',
         }}
       >
-        <Grid2 container spacing={0.5} justifyContent="center">
-          <Grid2 size={{ xs: 12, md: 6, xl: 3 }}>
-            <ConnectionOfTravelAssessmentCard assessment={connectionOfTravelAssessment} />
-          </Grid2>
-          <Grid2 size={{ xs: 12, md: 6, xl: 3 }}>
-            <StopLineStopAssessmentCard assessment={stopLineStopAssessment} />
-          </Grid2>
-          <Grid2 size={{ xs: 12, md: 6, xl: 3 }}>
-            <SignalStateEventAssessmentCard assessment={signalStateEventAssessment} />
-          </Grid2>
-          <Grid2 size={{ xs: 12, md: 6, xl: 3 }}>
-            <LaneDirectionOfTravelAssessmentCard assessment={laneDirectionOfTravelAssessment} />
-          </Grid2>
-          <Grid2 size={12}>
-            <NotificationsTable simple={true} />
+        <Container maxWidth={false} disableGutters>
```

Comment:
Question: What's the reasoning for the wrapping container element here?

Reply:
This is now not necessary - removed